backend/app/main.py

```diff
@@ -174,6 +174,26 @@ async def shatter_parent(
     return result
 
 
+@app.patch(
+    "/api/update_assignments/{document_id}/reset", status_code=status.HTTP_200_OK
+)
+async def reset_map(document_id: str, session: Session = Depends(get_session)):
```

Comment:
H: Please add a quick test for this function in `test_main.py`!

Reply:
Added in a0c76299654f8e8f28ecbafe0359440b21620beb !